recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e2ab66626fb57434ede7670ed8b2b2e8331919dbec78edb4d1d1646dfa0c73e7
+  sha256: 6c65e2122c8b62ee7985bb956369d05953c7704081f90b686a0fdfaf20933d2b
 
 build:
   number: 0
   #skip s390x as lack of nodejs and cartopy dependencies
-  skip: True  # [py<37 or s390x]
+  skip: True  # [py<310 or s390x]
 
```

Comment:
Could you try to add a generic section for `requirements` and `host`:
```
requirements:
  host:
    - python
    - pip
    - setuptools
    - wheel
```
I guess it can resolve the issue of `UnavailableInvalidChannel: HTTP 404 UNAVAILABLE OR INVALID for channel upload`

Reply:
Sorry, I don't understand what you're asking me to do. As far as I can see, both outputs have that (see below) so you must be asking for something else 🙃 

```yaml
    requirements:
      host:
        - python
        - pip
        - setuptools
        - wheel
```